src/models/sequence/sequence.go

```diff
@@ -0,0 +1,76 @@
+package sequence
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+)
+
+//Function Checkthedata validates the input data and ensures that it doesn;t contain any separators, symbols etc
+func Checkthedata(lenght string, minSq string) (lenght1 int, minSq1 int, err error) {
+	r := regexp.MustCompile("^[0-9]+$")
+	if r.MatchString(minSq) != true || r.MatchString(lenght) != true ||
+		r.MatchString(minSq) != true && r.MatchString(lenght) != true {
+		fmt.Println("Error. You entered a letter or a number with a separator.Program exited")
+		return 0, 0, errors.New("Input data contain symbols")
+	}
+
+	lenght1, _ = strconv.Atoi(lenght)
+	minSq1, _ = strconv.Atoi(minSq)
+	if lenght1 < 0 || minSq1 < 0 {
+		fmt.Println("You entered a negative number. Program excited")
+		return lenght1, minSq1, errors.New("Negative numbers used as an input data")
+	}
+	if lenght1 == 0 || minSq1 == 0 || lenght1 == 0 && minSq1 == 0 {
+		fmt.Println("Either of two values equals to 0")
+		return lenght1, minSq1, errors.New("One or both values is/are 0")
+	} else {
+		GetSquares(lenght1, minSq1)
+	}
+	return
```

Comment:
In fact I thought to do 2 thing at once. At first to prevent user from entering the symbols/separators etc using the general expression. But as far as I can see this doesn't solve the problem with entering negative numbers or numbers which are equal to 0. So I decided to convert the string into digit in the same function and compare them with 0.